### PR TITLE
Implement use of .env for database user

### LINKED ISSUE #1211: Implement use of .env for db username and passwd
## Little description
Given we are hardcoding database user as postgres and password as empty string in [database.yml](https://github.com/AgileVentures/LocalSupport/blob/b94d60ef45942dbca12925077deff0bfcee2da6a/config/database.yml#L7), if one has a username and password different from these they would be required to make the changes in the database.yml. 
It would be nice if these changes based on one's local environment are made on a gitignored .env file

### PR DESCRIPTION
**Pull request by aonomike**
- Reads the database username and password from environmental variables

#FIxes #1211

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ development: &dev
   database: ls_development
   host: <%= ENV['DB_HOST'] || ""%>
```
Comment (gabrielbaldao) [inline config/database.yml]: what do you think about add "localhost" as default to host?

File: config/database.yml.sample [hunk 2]
```diff
@@ -4,8 +4,8 @@ development: &dev
   database: ls_development
   host: <%= ENV['DB_HOST'] || ""%>
```
Comment (gabrielbaldao) [inline config/database.yml.sample]: same thing here.

File: config/database.yml (hunk 1 above)
Comment (aonomike) [reply config/database.yml]: yep makes sense, not sure why it wasnt set so 🤔

Comment (aonomike) [reply config/database.yml]: Done

File: config/database.yml.sample (hunk 2 above)
Comment (aonomike) [reply config/database.yml.sample]: Done

**Comment by aonomike** [comment]
@gabrielbaldao currently, the .env file is not gitignored, I am not sure if it would be the right file to hold these variables or if we want to gitignore it?

**Comment by Maroo-b** [comment]
@aonomike for ENV variables, it's common to provide a file named: `.env.sample` that contains all required ENV variables with random values. 

I noticed that we have `dotenv-rails` and `figaro` gems, let's remove `figaro` and keep `dotenv-rails` as it's more conveninet for use.

**Comment by aonomike** [comment]
@Maroo-b I have removed .env from being tracked by git and added the `.env.exampe` for guys to use for setup

**Pull request merged**